numpy/_core/tests/test_arrayprint.py

```diff
@@ -320,7 +320,7 @@ def test_structure_format_float(self):
         assert_equal(np.array2string(array_scalar), "(1., 2.12345679, 3.)")
 
     def test_unstructured_void_repr(self):
-        a = np.array([27, 91, 50, 75,  7, 65, 10,  8,
+        a = np.array([27, 91, 50, 75,   7, 65, 10,    8,
```

Comment:
🤔 
```suggestion
        a = np.array([27, 91, 50, 75, 7, 65, 10, 8,
```

Reply:
The idea is to have both arrays aligned. That's common practice in Numpy for tabular data.

Reply:
Yea it's pretty common. But IDE that are configured to automatically `ruff format` (which is pretty common in my experience) would reformat this either way, resulting in large diffs that the reviewers will have to go through (i.e. self-interest) :P.

Reply:
But I thought Numpy will not be formatted with ruff any time soon, precisely because of tabular data:
* https://github.com/astral-sh/ruff/discussions/7310

If so, any patch that reformats a file using `ruff format` will be rejected anyway, won't it?